asab/storage/elasticsearch.py

```diff
@@ -433,6 +573,13 @@ def __init__(self, storage, collection, obj_id, version=None):
 		if version == 0:
 			self.ModSet['_c'] = now  # Set the creation timestamp
 
+		self.SSLcontext = ssl.create_default_context(cafile=self.Storage.SSLcontext)
+
```

Comment:
In the StorgeService we created default ssl context as `self.SSLcontext = ssl.create_default_context(cafile=ssl_ca_file)`, where `ssl_ca_file `was a configuration item.

Here `self.Storage.SSLcontext` is used for a certificate path.